src/views/product-downloads-view/boundary/components/icon-linux-color-16.tsx

```diff
@@ -0,0 +1,61 @@
+/**
+ * Note: this should be replaced with an HDS component, once available.
+ * To check availability, search for "linux" at:
+ * https://flight-hashicorp.vercel.app/
+ */
+export function IconLinuxColor16() {
```

Comment:
This `linux` icon is needed, but not yet available in HDS Flight Icons. Have chatted with Brayden about this in Slack DMs. This will likely be added to HDS soon, but didn't want that to block this PR.